.gitignore

```diff
@@ -1,24 +1,12 @@
 .go-pkg-cache
 build
-dist
-venv
-env
+dist/*
 certs
 .idea
 *.pyc
-*~
 .coverage
-cover
-*.tar
 *.created-*
-docker
 **/*.sw[pon]
-birdcl
-.vagrant
-gobgp
-calicoctl/calicoctl
-bin
```

Comment:
And won't about bin and release? We still put files in those directories that we don't want checked in right?

Reply:
I don't think we put anything in either of those directories - looks like the binaries are built in `dist` and the release directory hasn't been used in a long time, AFAICT.